packages/react-scripts/config/config.js

```diff
@@ -0,0 +1,81 @@
+// @remove-on-eject-begin
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+// @remove-on-eject-end
+'use strict';
+
+const fs = require('fs');
+const chalk = require('chalk');
+const paths = require('./paths');
+
+/**
+ * Get the baseUrl of a compilerOptions object.
+ *
+ * @param {Object} options
+ */
+function getBaseUrl(options = {}) {
+  const baseUrl = options.baseUrl;
+
+  if (!baseUrl) {
+    return null;
+  }
+
+  if (baseUrl !== 'src' && baseUrl !== './src') {
```

Comment:
Should we restrict it to only `src` (or `./src`), or should we only say that `baseUrl` has to **start** with `src`? Maybe there is a need to have the `baseUrl` start from a directory nested inside `src`.

Reply:
Right now we only want to support `src`, if anyone has a good use case for setting the baseUrl to a nested directory we can always change it later.